operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -459,48 +458,48 @@ Review Release PRs and Community PRs to ensure the feature meets requirements fo
-* Add additional customer use cases to existing tickets. Encourage colleagues to add additional use cases to existing tickets as they gather information from their customer interactions. 
-* Link any implementation or design tickets to the request. 
-* Update the Fix Version of the feature to provide an indication of when the reporting request may be addressed. Generally, `Planned: Near Term` means it will be addressed in the next 1-3 releases, `Planned: Ongoing` means that is something that is being actively worked on, but progress may be slow, `Planned: Unscheduled` means we plan on addressing, but it is not currently on the roadmap, `Future Consideration` means we need to collect more details on use cases and demand before scheduling into a future roadmap and`Not Planned` means that we will not be working on addressing the request. 
-* Update the ticket status to closed when a feature is completed. 
+* Work with Customer Success and Sales to clarify use cases, root issues, and pain points as well as the priority of the request to the customer
+* Propose alternate solutions and workarounds when they are available
+* Provide feedback on whether it's a "Won't Do" feature that we are unlikely to add to the product
+* Update the feature idea to link any implementation or design tickets
+* Update the **Effort** for the feature with a SWAG from the Engineering team
+* Update the feature idea to have a release timeframe (3 months, 6 months, future) or release version if the feature is planned
+
+After adding the feedback to the relevant idea in productboard, the PM comments and mentions the reporter so they are aware the feedback has been processed, and then marks the item as **Processed**.
 
 #### Customer Request Rotation
 
-In sync with the Twitter rotation, Product Managers rotate responsibility weekly for monitoring newly filed [customer requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97&issueLimit=100) and ensuring newly filed requests are assigned to the appropriate PM according to areas of ownership. The weekly PM meeting is the time to identify requests where ownership is unlcear or merits discussion. The PM on rotation should also remind fellow PMs in the Product Management channel to review their requests and at a minimum update the Fix Version.
+In sync with the Twitter rotation, Product Managers rotate responsibility weekly for monitoring newly-filed [customer requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97&issueLimit=100) and ensuring newly filed requests are assigned to the appropriate PM according to areas of ownership. The weekly PM meeting is the time to identify requests where ownership is unlcear or merits discussion. The PM on rotation should also remind fellow PMs in the Product Management channel to review their requests and at a minimum update the Fix Version.
```

Comment:
we can remove this section, each PM now processes requests in Productboard so we no longer have the customer request rotation

Reply:
Removed, thanks. :)